src/Aspire.Hosting.Python/PythonAppResourceBuilderExtensions.cs

```diff
@@ -17,7 +17,7 @@ public static class PythonAppResourceBuilderExtensions
     /// </summary>
     /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/> to add the resource to.</param>
     /// <param name="name">The name of the resource.</param>
-    /// <param name="projectDirectory">The path to the directory containing the python app files.</param>
+    /// <param name="appDirectory">The path to the directory containing the python app files.</param>
     /// <param name="scriptPath">The path to the script relative to the project directory to run.</param>
```

Comment:
This still says project directory but I think it's referring to the python app directory?